python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -156,10 +169,16 @@ def dagit(
         os.environ["PYTHONWARNINGS"] = "ignore"
 
     configure_loggers()
-    logger = logging.getLogger("dagit")
+    logger = logging.getLogger("dagster-webserver")
```

Comment:
hmm, we may want to keep this logger name locked with the entry ie `dagit`, may break users log monitoring if we change it implicitly

Reply:
>keep this logger name locked with the entry ie dagit

What exactly do you mean by "name locked with the entry"?

If it's important to maintain the dagit logger, I see a couple options:

- Define both `dagster-webserver` and `dagit` loggers. Forward messages from `dagster-webserver` to `dagit`, and suppress `dagit` output when running `dagster dev`. This will create duplicate log messages in other channels.
- Define only `dagster-webserver` logger, but introduce custom formatting so that the log messages include "dagit" also-- something like:

```
2023-07-03 08:27:36 -0400 - dagster-webserver (dagit) - INFO - Serving dagster-webserver on http://127.0.0.1:3000 in process 78370
```

Just plain keeping it as "dagit" I think would be confusing to users who install "dagster-webserver" but then see "dagit" log messages.

Reply:
I just meant if a user invokes `dagit`  we name the logger `dagit` , similar to how we support `DAGIT_` env vars for that entry point

Reply:
OK, I just pushed a commit that modulates the logger name based on CLI command